releasenotes/notes/switch-to-qiskit-algorithms-b0af38fe547bf5b5.yaml

```diff
@@ -0,0 +1,24 @@
+---
+prelude: >
+    Qiskit Optimization 0.6 switches from ``qiskit.algorithms`` of Qiskit Terra
+    to `Qiskit Algorithms <https://github.com/qiskit-community/qiskit-algorithms>`_.
+    Qiskit Optimization 0.6 drops supports of the former algorithms based on
```

Comment:
Although support for the qiskit.algorithms primitive based ones is dropped, for the moment since they are compatible with what was migrated to qiskit_algorithms they should still work so any users code should still work as they transition/upgrade it - right.

Reply:
I'm wondering whether I have to mention qiskit.algorithms is not officially supported by qiskit-optimization but it works actually.

Reply:
The intent was really to have it work so users had a easier transition. Code that they had working based on primitives should still work with this new version. Then they can simply switch the code to qiskit_algorithms and it should work just like before.